### DISCUSSION TITLE
https://maxkim-j.github.io/posts/retrospect-running

### DISCUSSION THREAD

**Discussion by giscus** [MaxKim-J/maxkim-j.github.io#97] [Announcements]
# https://maxkim-j.github.io/posts/retrospect-running

많이도 달렸다

https://maxkim-j.github.io/posts/retrospect-running

**Comment by heka1024** [comment]
멋집니다!!!

**Comment by Johnyworld** [comment]
와우 종혁님 여전히 꾸준히 달리고 계시군요! 너무 멋있어요!! 리스펙 합니다 👍🏻
